src/Ubench.php

```diff
@@ -166,4 +166,12 @@ public static function readableElapsedTime($microtime, $format = null, $round =
 
         return sprintf($format, $time, $unit);
     }
+
+    public function wasEnd() {
```

Comment:
Hi, and thanks for your work.
Can you update to keep the same coding style as above ?

Reply:
@devster sorry, i have fixed it in tests but forget in source, updating ...